server/app.py

```diff
@@ -48,10 +48,10 @@ def create_app(csrf: CSRFProtect, db: SQLAlchemy):
     @app.route("/<path:path>")
     def serve(path):
         """ Registers paths required for serving frontend. """
-        if path != "" and os.path.exists(app.static_folder + "/" + path):
-            return send_from_directory(app.static_folder, path)
+        if path != "" and os.path.exists(app.static_folder + "/" + path):  # type: ignore
```

Comment:
A check for None should be added at the start of the function and raise a TypeError if app.static_folder is not set instead of disabling this warning